proj-xcode/Classes/networking/entity/PA2EncryptedRequest.h

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright 2017 Lime - HighTech Solutions s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "PA2NetworkObject.h"
+
+@class PA2ECIESEncryptor;
```

Comment:
Why do we need this class statement here? Seems unnecessary in the model class...

Reply:
Right, that's redundant. Previously, there was a reference to the `PA2ECIESCryptogram` & that enryptor, but I've changed that code. I'll fix this.